### PR TITLE
specs/atp: re-write specs intro

### PR DESCRIPTION
**Pull request by bnewbold**

### CODE REVIEW

File: content/specs/atp.md [hunk 1]
```diff
@@ -1,294 +1,82 @@
...
+**Network:** HTTP client-server and server-server [APIs](/specs/xrpc) are described with Lexicons, as are WebSocket [Event Streams](/specs/event-stream). Individual records can be referenced across the network by [AT URI](/specs/at-uri-scheme). A Personal Data Server (PDS) acts as an account's trusted agent in the network, routes client network requests, and hosts repositories. A Big Graph Server (BGS) crawls many repositories and outputs a unified event firehose.
 
-## Server-to-server API
+**Application:** APIs and record schemas for applications built on atproto are specified in [Lexicons](/specs/lexicon), which are are referenced by [Namespaced Identifiers](/specs/nsid) (NSIDs). Application-specific aggregations (such as search) are provided by an Application View (AppView) service. Clients can include mobile apps, desktop software, or web interfaces.
```
Comment (dholms) [inline content/specs/atp.md:23]:
```suggestion
**Application:** APIs and record schemas for applications built on atproto are specified in [Lexicons](/specs/lexicon), which are referenced by [Namespaced Identifiers](/specs/nsid) (NSIDs). Application-specific aggregations (such as search) are provided by an Application View (AppView) service. Clients can include mobile apps, desktop software, or web interfaces.
```

File: content/specs/atp.md [hunk 2]
```diff
@@ -1,294 +1,82 @@
...
 
-The server-to-server APIs enable federation, event delivery, and global indexing. They may also be used to provide application behaviors such as mail delivery and form submission.
+The AT Protocol itself does not specify common social media conventions like "follows" or "avatars", leaving these to application-level Lexicons. The `com.atproto.*` Lexicons provide common APIs for things like account signup and login. These could be considered part of AT Protocol itself, though they can also be extended or replaced over time as needed. Bluesky is a Twitter-style microblogging application built on top of AT Protocol, with lexicons under the `app.bsky.*` namespace.
+
+## Cryptography
```
Comment (dholms) [inline content/specs/atp.md:27]: kinda feel that this could warrant it's own doc

Comment (dholms) [reply content/specs/atp.md:27]: gets a bit in the weeds for the overview docs

Comment (bnewbold) [reply content/specs/atp.md:27]: @dholms: I felt like it was important to at least mention these at a high-level in the first page, to clarify the scope of these spec docs.

We will definitely need some additional docs for both the com.atproto "built-ins" and the app.bsky application stuff. They are only partially self-documenting via Lexicon.

File: content/specs/atp.md [hunk 3]
```diff
@@ -1,294 +1,82 @@
...
 
-- [com.atproto.identity](/lexicons/com-atproto-identity). Handle resolution and changes.
-- [com.atproto.server](/lexicons/com-atproto-server). Account and session management.
-- [com.atproto.moderation](/lexicons/com-atproto-moderation). Moderation reporting.
-- [com.atproto.repo](/lexicons/com-atproto-repo). Repo CRUD operations.
-- [com.atproto.sync](/lexicons/com-atproto-sync). Repo content sync and streaming.
+**Identity:** account control is rooted in stable [DID](/specs/did) identifiers, which can rapidly resolved to determine the current service provider location and cryptographic keys associated with the account. [Handles](/specs/handle) provide a more human-recognizable and mutable identifier for accounts.
 
-### Additional lexicons
+**Data:** public content is stored in content-addressed and cryptographically verifiable [Repositories](/specs/repository). data records and network messages all conform to a unified [Data Model](/specs/data-model) ([IPLD](https://ipld.io/docs/data-model/), with [CBOR](https://en.wikipedia.org/wiki/CBOR) and JSON representations).
```
Comment (emilyliu7321) [inline content/specs/atp.md:19]:
```suggestion
**Data:** public content is stored in content-addressed and cryptographically verifiable [Repositories](/specs/repository). Data records and network messages all conform to a unified [Data Model](/specs/data-model) ([IPLD](https://ipld.io/docs/data-model/), with [CBOR](https://en.wikipedia.org/wiki/CBOR) and JSON representations).
```

File: content/specs/atp.md (hunk 1 above)
Comment (emilyliu7321) [reply content/specs/atp.md:23]:
```suggestion
**Application:** APIs and record schemas for applications built on atproto are specified in [Lexicons](/specs/lexicon), which are referenced by [Namespaced Identifiers](/specs/nsid) (NSIDs). Application-specific aggregations (such as search) are provided by an Application View (App View) service. Clients can include mobile apps, desktop software, or web interfaces.
```

File: content/specs/atp.md [hunk 4]
```diff
@@ -1,294 +1,82 @@
...
 
-## Server-to-server API
+**Application:** APIs and record schemas for applications built on atproto are specified in [Lexicons](/specs/lexicon), which are are referenced by [Namespaced Identifiers](/specs/nsid) (NSIDs). Application-specific aggregations (such as search) are provided by an Application View (AppView) service. Clients can include mobile apps, desktop software, or web interfaces.
 
-The server-to-server APIs enable federation, event delivery, and global indexing. They may also be used to provide application behaviors such as mail delivery and form submission.
+The AT Protocol itself does not specify common social media conventions like "follows" or "avatars", leaving these to application-level Lexicons. The `com.atproto.*` Lexicons provide common APIs for things like account signup and login. These could be considered part of AT Protocol itself, though they can also be extended or replaced over time as needed. Bluesky is a Twitter-style microblogging application built on top of AT Protocol, with lexicons under the `app.bsky.*` namespace.
```
Comment (emilyliu7321) [inline content/specs/atp.md:25]:
```suggestion
The AT Protocol itself does not specify common social media conventions like follows or avatars, leaving these to application-level Lexicons. The `com.atproto.*` Lexicons provide common APIs for things like account signup and login. These could be considered part of AT Protocol itself, though they can also be extended or replaced over time as needed. Bluesky is a microblogging social app built on top of AT Protocol, with lexicons under the `app.bsky.*` namespace.
```

File: content/specs/atp.md [hunk 5]
```diff
@@ -1,294 +1,82 @@
...
+
+The set of supported cryptographic systems is expected to evolve slowly. There are significant interoperability and implementation advantages to having as few systems as possible at any point in time.
+
+## Protocol Extension and Applications
+
+AT Protocol was designed from the beginning to balance stability and interoperation against flexibility for third-party application development.
+
+The core protocol extension mechanism is development of new Lexicons under independent namespaces. Lexicons can declare new repository record schemas (stored in collections by NSID), new HTTP API endpoints, and new event stream endpoints and message types. It is also expected that new applications might require new network aggregation services ("AppViews") and client apps (eg, mobile apps or web interfaces).
+
+It is expected that third parties will re-use Lexicons and record data across namespaces. For example, new applications are welcome to build on top of the social graph records specified in the `app.bsky.*` Lexicons, as long as they comply with the schemas controlled by the `bsky.app` authority.
```
Comment (emilyliu7321) [inline content/specs/atp.md:52]:
```suggestion
It is expected that third parties will reuse Lexicons and record data across namespaces. For example, new applications are welcome to build on top of the social graph records specified in the `app.bsky.*` Lexicons, as long as they comply with the schemas controlled by the `bsky.app` authority.
```

**Comment by bnewbold** [comment]
TODO(bnewbold): https://bsky.app/profile/retr0-id.translate.goog/post/3jybyf5i7yo2s

"high-S" secp256k1 signatures, BIP-62

File: content/specs/atp.md (hunk 2 above)
Comment (bnewbold) [reply content/specs/atp.md:27]: oh, I see you meant the Cryptography section (github PR review page wasn't clear). Yeah, cryptography could be a separate page.

**Comment by bnewbold** [comment]
Rolled-up a couple other small corrections from feedback in other venues (slack, in-app, etc). Larger changes i'll do as separate PRs.

**Pull request merged**